core/src/Settings/Configuration/PayPalIntentConfiguration.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace PsCheckout\Core\Settings\Configuration;
+
+class PayPalIntentConfiguration
```

Comment:
I really dislike the "Configuration" part of the name, because it's more of an enum